### ISSUE TITLE
explore timeline tracking solution

### ISSUE THREAD

**Issue by altsalt** [SeaGL/organization#378]
Does not need to be tied to current workflow. Should try to avoid having to register with an additional service. Burndown charts and Kanban boards are effective, can they be combined? Linking to GH issues is fine, but we need some way to assign multiple people to a task that has a deadline and some view of what items are past-due.

**Comment by altsalt** [comment]
I just noticed that hedgedoc supports [gantt charts](https://demo.hedgedoc.org/features#Mermaid). This would provide a nice visual assistance, but still means due dates are checked manually and not linked to any sort of kanban card or issue tracker.

**Comment by strugee** [comment]
IIRC we also discussed just deploying https://github.com/jeobz/NxDeckGantt as a visualization tool on top of Nextcloud Deck, which of course we already have access to

**Comment by strugee** [comment]
(Nobody AFAIK has tried this though so it might totally not work for us)

**Comment by altsalt** [comment]
Yeah, that's the next thing to try afaik and I hope it works, just wanted to note another option here :)
